src/main/java/Event.java

```diff
@@ -0,0 +1,22 @@
+public class Event extends Task{
+    
+    protected String at;
+
+    public Event(String name, String at) {
+        super(name);
+        this.at = at;
+    }
+
+    public void setAt(String at) {
```

Comment:
I may be wrong, but I don't see the setters and getters methods being called, so maybe they could be ommitted?